docs/_static/custom.js

```diff
@@ -19,10 +19,24 @@ function openModal(modal) {
   modal.style.removeProperty('display');
 }
 
+function updateSetting(element) {
+  localStorage.setItem(element.name, element.checked);
```

Comment:
I feel like we should prefix the storage keys here with something unique, maybe `discordpyDocs_` or something?

Reply:
This'll probably call for another function to grab settings though, since repeating the key prefix everywhere wouldn't be ideal.